content/en/blog/2025/devex-survey/index.md

```diff
@@ -0,0 +1,148 @@
+    dropping data. Better tooling for local debugging would help with
+    troubleshooting.
+- SDK configuration:
+  - Overall complexity (both configuration and concepts)
+  - SDK configuration is complex, and many users have pointed out that
+    troubleshooting the exporter is the most painful part.
+- Collector operations:
+  - Debugging guidance on the Collector configuration and its components is
+    needed. OTTL was mentioned a couple of times for being difficult to
+    troubleshoot.
+
+## Detailed Insights
+
+### General Information about Respondents
+
+The survey responses came from users, of whom the majority use OpenTelemetry in
+production (83%) and do not work for a vendor (77.4%). 82% consider their
+organization to be at an intermediate or advanced phase of their journey with
+observability. The platforms used leaned towards Kubernetes (68.2%) and AWS
+(34.6%), with bare metal still making up a respectable percentage (12%).
+Programming language usage was split between the usual suspects of Go, Java,
+JavaScript, Python and .NET, but had a wide spread and even languages without
+official API/SDK implementations, as shown in the second graph below. While
+those working on backends or operations were the majority of respondents, mobile
+and browser were represented as well, see the third graph:
+
+![Forms response chart. Question title: Where in your Observability journey is
+your organization?. Number of responses: 217 responses.](image1.png)
+
+![](image2.png)
+
+![Forms response chart. Question title: What kind of service do you regularly
+work on? (select all that apply). Number of responses: 217
+responses.](image3.png)
+
+### Documentation
+
+Results showed that, by far, people were getting their day one information when
+getting started with OpenTelemetry from the website and GitHub repositories for
+languages, 75.1% and 43.8%, while 25.1% were starting from their vendor’s
+documentation. However, unclear documentation took the top spot in the three
+questions related to getting started with manual instrumentation, auto
+instrumentation and setting up the SDK. In the collector's case, it took second
+place to “configuring the processors”.
+
+More details on the user experience with documentation can be found in the
+survey results of the [survey](/blog/2024/otel-docs-survey/) conducted by the
+End User SIG.
+
+### Collector Usage
+
+Getting started with the Collector was shown to lean towards the fairly easy,
+though based on results for the question asking for problems with operating the
+Collector and the overall experience question, it seems clear there is a gap between
+getting started and operating with more complex pipelines or in larger deployments,
+more on this in the Future Plans section.
+
+Interestingly, _collector-contrib_ had a large percentage of the responses to
+the question, “Which collector do you use?”. Sadly, it is unclear if this is due
```

Comment:
Why "Sadly"? We might to have explain a bit more about why the OCB should be the happy path.

Reply:
Here the sadly was referring to simply that the survey wasn't able to surface that kind of detail. But it is true we are also implying it is better to use ocb than contrib so should have an explanation. I originally was going to include that but it gets long and complicated, plus  veers off topic.

Let me see if I can find a good short explanation from the OCB docs. I also didn't want to imply too much about implications of using contrib since we haven't gone to the Collector SIG with this yet to discuss.

Reply:
Sorry, didn't report back that I checked the OCB docs and they give the impression OCB is for if you have a custom component, so I could not find a short explanation from there.